osu.Game.Rulesets.Taiko/Objects/TaikoStrongableHitObject.cs

```diff
@@ -17,7 +17,7 @@ public abstract class TaikoStrongableHitObject : TaikoHitObject
         /// <summary>
         /// Scale multiplier for a strong drawable taiko hit object.
         /// </summary>
-        public const float STRONG_SCALE = 1.4f;
+        public const float STRONG_SCALE = 1.525f;
```

Comment:
If we want to make this perfectly matching:

```suggestion
        public const float STRONG_SCALE = 1 / 0.65f;
```

Haven't confirmed visually so please double-check.

Reply:
Matches better than the current ratio on quick visual comparisons.